SteamKit2/SteamKit2/Networking/Steam3/UdpConnection.cs

```diff
@@ -356,22 +366,49 @@ private bool DispatchMessage()
         /// <summary>
         /// Processes incoming packets, maintains connection consistency, and oversees outgoing packets.
         /// </summary>
-        private void NetLoop()
+        private void NetLoop(object param)
         {
             // Variables that will be used deeper in the function; locating them here avoids recreating
             // them since they don't need to be.
-            EndPoint packetSender = (EndPoint) new IPEndPoint(IPAddress.Any, 0);
+            var userRequestedDisconnect = false;
+            EndPoint packetSender = (EndPoint)new IPEndPoint(IPAddress.Any, 0);
             byte[] buf = new byte[2048];
 
-            timeOut = DateTime.Now.AddSeconds(TIMEOUT_DELAY);
-            nextResend = DateTime.Now.AddSeconds(RESEND_DELAY);
+            if (param is IPEndPoint)

```

Comment:
I believe this is a mistake, as I don't think `param` can even be an `IPEndPoint` so this can be simplified to a single `as`